docs/core/whats-new/dotnet-core-3-0.md

```diff
@@ -311,9 +258,23 @@ The [Windows Application Packaging Project](https://docs.microsoft.com/windows/u
 > <RuntimeIdentifiers>win-x86;win-x64</RuntimeIdentifiers>
 > ```
 
+## WinForms HighDPI
+
+.NET Core Windows Forms applications can set High DPI mode with <xref:System.Windows.Forms.Application.SetHighDpiMode(System.Windows.Forms.HighDpiMode)>. The `SetHighDpiMode` method sets the corresponding High DPI mode unless the setting has been set by other means like `App.Manifest` or P/Invoke before `Application.Run`.
+
+The possible `highDpiMode` values, as expressed by the <xref:System.Windows.Forms.HighDpiMode> enum are:
+
+* `DpiUnaware`
+* `SystemAware`
+* `PerMonitor`
+* `PerMonitorV2`
+* `DpiUnawareGdiScaled`
+
+For more information about High DPI modes, see [High DPI Desktop Application Development on Windows](/windows/desktop/hidpi/high-dpi-desktop-application-development-on-windows).
+
 ## Fast built-in JSON support
 
-The .NET ecosystem has relied on [**Json.NET**](https://www.newtonsoft.com/json) and other popular JSON libraries, which continue to be good choices. **Json.NET** uses .NET strings as its base datatype, which are UTF-16 under the hood.
+The .NET ecosystem has relied on [**Json.NET**](https://www.newtonsoft.com/json) and other popular JSON libraries, which continue to be good choices. **Json.NET** uses .NET strings as its base datatype, which is UTF-16 under the hood.
 
 The new built-in JSON support is high-performance, low allocation, and based on `Span<byte>`. Three new main JSON-related types have been added to .NET Core 3.0 the `System.Text.Json` namespace.
 
```

Comment:
Line 326 talks about Preview 1. I believe the status has changed since then and coming next should be reviewed. @ahsonkhan can probably comment on this section.